### PR TITLE
Update limit-storage-consumption.md for zh-cn

### PR DESCRIPTION
**Pull request by chymy**
StorageQuota-->ResourceQuota

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/limit-storage-consumption.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ AWS EBS volumes have a 1Gi minimum requirement.
 <!--
 ## StorageQuota to limit PVC count and cumulative storage capacity
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/limit-storage-consumption.md:98]:
```suggestion
## ResourceQuota to limit PVC count and cumulative storage capacity
```

**Review by chymy — COMMENTED** [review]
update commend

File: content/zh-cn/docs/tasks/administer-cluster/limit-storage-consumption.md (hunk 1 above)
Comment (chymy) [reply content/zh-cn/docs/tasks/administer-cluster/limit-storage-consumption.md]: thanks, done.

**Comment by windsonsea** [comment]
Thanks, please squash your commits into a single one. Refer to:
https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

**Comment by chymy** [comment]
> Thanks, please squash your commits into a single one. Refer to: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

I have reopened a PR #47144 

/close

**Pull request closed without merging**